### PR TITLE
global reader app permission policies visibility

### PR DESCRIPTION
**Pull request by grabery**
Mention about the missing option and visibility of app permission policies, if you're not a global admin nor teams admin.

### CODE REVIEW

File: Teams/teams-app-permission-policies.md [hunk 1]
```diff
@@ -27,6 +27,9 @@ ms.custom:
 
 As an admin, you can use app permission policies to control what apps are available to Microsoft Teams users in your organization. You can allow or block all apps or specific apps published by Microsoft, third-parties, and your organization. When you block an app, users who have the policy are unable to install it from the Teams app store. You must be a global admin or Teams service admin to manage these policies.
 
+> [!NOTE]
+> If you are not a global admin or Teams service admin, you will not be able to see the app permission policies in the admin portal. Even if you do have the global reader role assigned.
```
Comment (JohanFreelancer9) [inline Teams/teams-app-permission-policies.md:31]:
```suggestion
> If you are not a global admin or Teams service admin, you will not be able to see the app permission policies in the admin portal, even if you do have the global reader role assigned.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @grabery, kindly commit the suggested change. Many thanks.

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by grabery** [comment]
missed my mention

**Pull request closed without merging**